packages/garbo/src/resources/darts.ts

```diff
@@ -0,0 +1,52 @@
+import { $effect, $item, get, have, maxBy } from "libram";
+
+export const guaranteedBullseye = () =>
+  get("everfullDartPerks").includes("25% Better bullseye targeting") &&
+  get("everfullDartPerks").includes("25% More Accurate bullseye targeting") &&
+  get("everfullDartPerks").includes("25% better chance to hit bullseyes");
+
+export const safeToAttemptBullseye = () =>
+  have($item`Everfull Dart Holster`) &&
+  (guaranteedBullseye() || have($item`spring shoes`)) &&
+  !dartLevelTooHigh();
+
+export const canBullseye = () =>
+  have($effect`Everything Looks Red`) &&
```

Comment:
```suggestion
  !have($effect`Everything Looks Red`) &&
```

Isn't this backwards?